pallets/schemas/src/migration/v5.rs

```diff
@@ -0,0 +1,184 @@
+use crate::{
+	migration::v4,
+	pallet::{CurrentIntentIdentifierMaximum, IntentInfos, NameToMappedEntityIds, SchemaInfos},
+	Config, IntentInfo, Pallet, SchemaInfo, SchemaVersionId, SCHEMA_NAME_BYTES_MAX,
+	SCHEMA_STORAGE_VERSION,
+};
+use alloc::{format, vec::Vec};
+use common_primitives::schema::{IntentId, MappedEntityIdentifier, SchemaId, SchemaStatus};
+use core::marker::PhantomData;
+#[cfg(feature = "try-runtime")]
+use frame_support::ensure;
+use frame_support::{
+	pallet_prelude::{Get, GetStorageVersion, Weight},
+	traits::{Len, UncheckedOnRuntimeUpgrade},
+	BoundedVec,
+};
+#[cfg(feature = "try-runtime")]
+use parity_scale_codec::{Decode, Encode};
+#[cfg(feature = "try-runtime")]
+use sp_runtime::TryRuntimeError;
+
+const LOG_TARGET: &str = "pallet::schemas::migration::v5";
+
+fn convert_from_old(id: SchemaId, old_info: &v4::SchemaInfo) -> crate::SchemaInfo {
+	SchemaInfo {
+		intent_id: id as IntentId,
+		model_type: old_info.model_type,
+		payload_location: old_info.payload_location,
+		settings: old_info.settings,
+		status: SchemaStatus::Active,
+	}
+}
+
+fn convert_to_intent(old_info: &v4::SchemaInfo) -> IntentInfo {
+	IntentInfo { payload_location: old_info.payload_location, settings: old_info.settings }
+}
```

Comment:
nit: can you implement `Into` trait

Reply:
Right--I thought about it, but figured since it's a migration that will eventually be deleted, I would rather have it all be self-contained.